CHANGES.rst

```diff
@@ -23,8 +23,8 @@ Release History
 
 **Added**
 
-- Added slack notifications for failing master builds.
-  (`#65 <https://github.com/nengo/nengo-fpga/pull/65>`__)
```

Comment:
Just removing this because it doesn't concern the user?

Reply:
Yeah. Trevor informed me that the changelog should just be user facing changes, so I took the opportunity to remove the slack notification one.